src/main/java/com/internet/shop/Main.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.shop;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+
+public class Main {
+   private static Injector injector = Injector.getInstance("com.internet.shop");
```

Comment:
```suggestion
    private static Injector injector = Injector.getInstance("com.internet.shop");
```
now that you have maven checkstyle plugin, you can run `mvn clean package` to check codestyle